### DISCUSSION TITLE
How good is this compare to IDM  ?

### DISCUSSION THREAD

**Discussion by DareDevil2000** [amir1376/ab-download-manager#29] [Q&A]
I would like to hear from people who have used this utility for a while.

**Comment by zlf179** [comment]
cannot download youtube videos by sniffer like the function on IDM
But the UI is much better than IDM

**Comment by parag60288** [comment]
Apart from video sniffing, everything else is as good as (sometimes better than) IDM.

**Comment by armindeck** [comment]
Es una de las mejores alternativas que existen, yo use IDM por un largo tiempo.
Hace varios meses empecé a buscar alternativas, entre esas Free Download Manager, pero tuve problemas con las velocidades de descarga, por lo que no me termino gustando, y mas la versión de linux, pero cuando conoci a AB Download Manager me quedo gustando ya que cumple con casi todas las funciones de IDM.

**Reply by ebrahim-zarei** [reply]
Best alternative to IDM , but need to addon for downloading youtube video

**Reply by armindeck** [reply]
Es cierto, todavía falta mejorar la extensión para poder descargar videos sin necesidad de copiar los enlaces, y no solo en YouTube.

**Comment by ero35** [comment]
New Feature Suggestion;
1-Capture audio and video files
2-BitTorrent, Magnet for Download.

I use ant downlod maneger
https://antdownloadmanager.com/index

**Reply by Patrxgt** [reply]
Nah, you should use proper torrent downloader to download files from torrents. Instead, I would suggest adding an option to redirect any .torrent file directly to system default torrent downloader instead of downloading it like any other file.

**Comment by spidy2356** [comment]
plz add video/audio sniffer from various sites, it's too much helpful

**Comment by LiveOnlyFans** [comment]
it better and dont ask you about license every time you use it.
